packages/extension-browser/scripts/pack-mdn-data.js

```diff
@@ -1,13 +1,245 @@
+ * @returns {boolean}
+ */
+const isFeatureUniversallySupported = (compat) => {
+    for (const browserName in compat.support) {
+        for (const supportStatement of getSupportStatements(browserName, compat.support)) {
+            if (!isUniversalSupportStatement(browserName, supportStatement)) {
+                return false;
+            }
+        }
+    }
+
+    return true;
+};
+
+/**
+ * Determine if a feature is *not* supported anywhere.
+ *
+ * @param {bcd.CompatStatement} compat
+ */
+const isFeatureUniversallyUnsupported = (compat) => {
+    for (const browserName in compat.support) {
+        const supportStatements = getSupportStatements(browserName, compat.support);
+
+        if (supportStatements.length > 1 || supportStatements[0].version_added) {
+            return false;
+        }
+    }
+
+    return true;
+};
+
+/**
+ * Remove any `bcd.SimpleSupportStatement` that requires `flags`.
+ * Flagged entries are ignored since they must be enabled by end users.
+ *
+ * @param {string} browserName
+ * @param {bcd.SupportBlock} support
+ */
+const removeFlaggedSupport = (browserName, support) => {
+    const supportStatements = getSupportStatements(browserName, support);
+    const unflaggedSupportStatements = supportStatements.filter((supportStatement) => {
+        return !supportStatement.flags;
+    });
+
+    if (unflaggedSupportStatements.length) {
+        support[browserName] = unflaggedSupportStatements;
+    } else {
+        delete support[browserName];
+    }
+};
+
+/**
+ * Remove all notes as they are not useful for analysis.
+ *
+ * @param {string} browserName
+ * @param {bcd.SupportBlock} support
+ */
+const removeSupportNotes = (browserName, support) => {
+    for (const supportStatement of getSupportStatements(browserName, support)) {
+        delete supportStatement.notes;
+    }
+};
+
+/**
+ * Remove all support statements which indicate universal support.
+ *
+ * @param {string} browserName
+ * @param {bcd.SupportBlock} support
+ */
+const removeUniversalSupport = (browserName, support) => {
+    for (const supportStatement of getSupportStatements(browserName, support)) {
+        if (isUniversalSupportStatement(browserName, supportStatement)) {
+            delete support[browserName];
+
+            return;
+        }
+    }
+};
+
+/**
+ * Remove all data about a feature which is unnecessary for analysis.
+ *
+ * @param {bcd.Identifier} data
+ */
+const removeFeatureData = (data) => {
+    if (!data.__compat) {
+        return;
+    }
+
+    const compat = data.__compat;
+    const support = data.__compat.support;
+
+    // Remove unnecessary data cross-browser.
+    if (isFeatureUniversallySupported(compat) || isFeatureUniversallyUnsupported(compat)) {
+        delete data.__compat;
+
+        return;
+    }
+
+    // Description is not needed for analysis.
+    delete compat.description;
+
+    // Remove unnecessary data per-browser.
+    for (const browserName in support) {
```

Comment:
Why don't you use `Object.entries(support)` or `Object.keys(support)` so you don't need to use a `for .. in` to iterate?

Reply:
No particular reason, just habit 😊 I'll switch over and update the PR.

Reply:
Done!